rpc_proxy.go

```diff
@@ -307,9 +307,11 @@ func (p *rpcProxy) director(ctx context.Context,
 
 	outCtx := metadata.NewOutgoingContext(ctx, mdCopy)
 
-	// Is there a basic auth set?
+	// Is there a basic auth or super macaroon set?
 	authHeaders := md.Get("authorization")
-	if len(authHeaders) == 1 {
+	macHeader := md.Get(HeaderMacaroon)
+	switch {
+	case len(authHeaders) == 1:
 		macBytes, err := p.basicAuthToMacaroon(
```

Comment:
doesnt `basicAuthToMacaroon` possibly add a supermac that we then also need to convert?

Reply:
Yes, you're correct! But only in `lnd` integrated mode, in which every daemon also is integrated and we actually _can_ use the super macaroon. At least I think we decided to declare the modes "lnd integrated but faraday/loop/pool remote" as un-supported. Perhaps we should document this... Yeah, going to add another compatibility matrix for this...